src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
+  }, 3000);
+});
+
+const successHandler = () => {
+  const successMessage = document.createElement('div');
+
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.appendChild(successMessage);
+};
+
+const errorHandler = () => {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
+};
+
+promise1
+  .then(successHandler)
+
+  .catch(errorHandler);
+
+promise2
+  .then(successHandler)
+
+  .catch(errorHandler);
```

Comment:
The promise2 is set to reject after 3 seconds, triggering the errorHandler. However, if you want to handle both promises together, consider using Promise.all or Promise.race to manage their outcomes collectively.